Client/Application/NavigationRouter.swift

```diff
@@ -55,7 +56,7 @@ enum DeepLink {
             self = .homePanel(link)
         } else if component == "default-browser", let link = DefaultBrowserPath(rawValue: String(componentPath)) {
             self = .defaultBrowser(link)
-        } else if component == "join" {
+        } else if component == "invite" {
```

Comment:
I just had the idea to make the string "invite" a constant somewhere - maybe in Referrals or in Environment. Just an idea, no blocker.